src/main/java/ledger/user_interface/ui_controllers/TransactionPopupController.java

```diff
@@ -130,22 +131,32 @@ private void printStackTrace(Exception e) {
      * @return a new Transaction object consisting of user input
      */
     public Transaction getTransactionSubmission() {
-        try {
+
             LocalDate localDate = this.datePicker.getValue();
             Instant instant = Instant.from(localDate.atStartOfDay(ZoneId.systemDefault()));
             this.date = Date.from(instant);
 
             this.pending = this.pendingCheckBox.isSelected();
 
             this.payee = fromBoxPayee(this.payeeText.getValue());
+            if (InputSanitization.isInvalidPayee(this.payee)){
+                this.setupErrorPopup("Invalid Payee entry.", new Exception());
```

Comment:
Should this take just the generic Exception with it? Or should it be something more specific?